mfs/ops.go

```diff
@@ -104,7 +105,7 @@ func PutNode(r *Root, path string, nd node.Node) error {
 
 // Mkdir creates a directory at 'path' under the directory 'd', creating
 // intermediary directories as needed if 'mkparents' is set to true
-func Mkdir(r *Root, pth string, mkparents bool, flush bool) error {
+func Mkdir(r *Root, pth string, mkparents bool, flush bool, cidVer dag.CidVersion) error {
```

Comment:
Lets have this be a field on the `Root` object, that way we can avoid changing the mfs APIs